crates/metaslang/bindings/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+version.workspace = true
+rust-version.workspace = true
+edition.workspace = true
+publish = true
+
+name = "metaslang_bindings"
```

Comment:
I suggest adding the new crate to `USER_FACING_CRATES`, otherwise, it would fail publishing:

https://github.com/manastech/slang/blob/4b2c495d036c1ab81b36ef7c2dccf0f097780235/crates/infra/cli/src/commands/publish/cargo/mod.rs#L14-L19